Documentation/workflow/OfficalAndDailyBuilds.md

```diff
@@ -12,11 +12,36 @@ and you can expect to see new versions roughly three months.   However it is als
 Team publishes **daily builds** of all sorts of packages including those built by the CoreCLR and CoreFX 
 repositories.  You can see what is available from
 
- * <https://dotnet.myget.org/gallery/dotnet-core>, and in particular you can see the builds of CoreCLR at 
- * <https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.NETCore.Runtime.CoreCLR>.   
- 
+ * <https://dotnet.myget.org/gallery/dotnet-core>, and in particular you can see the builds of 
+ * CoreCLR at <https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.NETCore.Runtime.CoreCLR>
+ * NETCore.App at <https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.NETCore.App>
+
 Thus if your goal is just to get the latest bug fixes and features, you don't need to build CoreCLR yourself you 
-can simply add <https://dotnet.myget.org/F/dotnet-core/api/v3/index.json> to your Nuget Feed list. 
+can simply add <https://dotnet.myget.org/F/dotnet-core/api/v3/index.json> to your Nuget Feed list and set the
+`RuntimeFrameworkVersion` in your project file to a `Microsoft.NETCore.App` version. You need to restore
+and publish your application so it includes the runtime (`self-contained`).
+
+```
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+ <packageSources>
+    <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json"/>
+ </packageSources>
+</configuration>
+```
+```
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <RuntimeFrameworkVersion>2.0.0-preview2-*/RuntimeFrameworkVersion>
+  </PropertyGroup>
+</Project>
+```
+```
+$ dotnet restore -r linux-x64
+$ dotnet publish -r linux-x64 --self-contained true
```

Comment:
I thought that this was because of a bug, and it would eventually get fixed. Also, when I recently did this, I didn't pass in the `--self-contained` argument to the `dotnet` command, but instead set a property on my csproj. You can see an example [here](https://github.com/dotnet/corefx/blob/master/src/SharedFrameworkValidation/RestoreSDKProject/RestoreSDKProject.csproj#L8)
```xml
    ...
    <RuntimeIdentifier>$(_PublishRID)</RuntimeIdentifier>
    <SelfContained>true</SelfContained>
  </PropertyGroup>
  ...
```